osmaxx/utils/polyfile_helpers.py

```diff
@@ -1,6 +1,24 @@
 import os
+from django.contrib.gis.geos import Polygon, MultiPolygon, GEOSGeometry
 
-from django.contrib.gis.geos import MultiPolygon, Polygon
+from osmaxx.countries._settings import POLYFILE_LOCATION
+
+POLYFILE_ENDING = '.poly'
```

Comment:
Maybe name this `POLYFILE_FILENAME_EXTENSION` to distinguish it from the ending of polygon filter file format file contents.